include/deal.II/numerics/time_dependent.h

```diff
@@ -524,7 +525,7 @@ class TimeDependent
    * the TimeStepBase class.
    *
    * Instead of using the above form, you can equally well use
-   * <tt>bind2nd(mem_fun1(&X::unary_function), arg)</tt> which lets the @p
+   * <tt>std_cxx11::bind(&X::unary_function, objX, arg)</tt> which lets the @p
```

Comment:
I believe I misunderstood the intended use. This has been revised to a working approach.